sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/AuthenticationEventResponseHandler.cs

```diff
@@ -93,13 +94,20 @@ public Task SetValueAsync(object result, CancellationToken cancellationToken)
 
                 if (Response.StatusCode == System.Net.HttpStatusCode.OK)
                 {
-                    Response.Validate();
-                    Response.Invalidate();
+                    try
+                    {
+                        Response.Validate();
+                        Response.Invalidate();
+                    }
+                    catch (ValidationException exception)
+                    {
+                        throw new AuthenticationEventTriggerResponseValidationException(exception.Message);
+                    }
```

Comment:
Move this logic inside of `AuthenticationEventResponse.Validation()`

`AuthenticationEventResponse.Validation()` should know to throw the correct exception `AuthenticationEventTriggerResponseValidationException`